apps/src/dance/ai/utils/chooseEffects.ts

```diff
@@ -23,7 +33,9 @@ const NUM_RANDOM_BOTTOM_OPTIONS = 20;
  */
 export function chooseEffects(
   selectedEmojis: string[],
-  quality: EffectsQuality
+  quality: EffectsQuality,
+  calculateSummedWeights: CalculateSummedWeights = calculateOutputSummedWeights,
+  numOptions: number | undefined = undefined
```

Comment:
What do you think of using `numOptions?: number` instead of `| defined = undefined`?

Reply:
Much better, thanks! Will update